extensions/kafka-connect/src/test/java/com/hazelcast/jet/kafka/connect/KafkaConnectDatagenIntegrationTest.java

```diff
@@ -87,5 +93,98 @@ public void testReadFromDatagenConnector() throws Exception {
         }
     }
 
+    @Test
+    public void testScalingWithDatagenConnector() throws Exception {
```

Comment:
datagen is already in the class name

Reply:
renamed: https://github.com/hazelcast/hazelcast/pull/23831/commits/b10441acecac5e8713d08f1c0c625f31effa7d09